quickevent/app/quickevent/plugins/Receipts/src/receiptsplugin.cpp

```diff
@@ -184,108 +184,163 @@ QVariantMap ReceiptsPlugin::receiptTablesData(int card_id)
 	bool is_card_lent = getPlugin<CardReaderPlugin>()->isCardLent(card_id, read_card.finishTime(), run_id);
```

Comment:
It is not directly related to this PR but IMO this does not work properly. First argument of function isCardLent should SI card number, not id from cards table.

Reply:
afaik `isCardLent` takes id, not card number, `card_id` is an abbreviation of `id` form table `cards`
 it surely does not help that si card number is named `siId` in the db schema (the table `cards` contains both `id` and `siId` columns)

Reply:
There is a query in `isCardLent` where is surely card number expected:
`q.exec("SELECT siId FROM lentcards WHERE NOT ignored AND siid=" QF_IARG(si_id) );`

and also `findRunId` is called which also takes card number